src/garage/torch/algos/maml.py

```diff
@@ -419,34 +467,30 @@ class MAMLTrajectoryBatch(
     +=======================+=================================================+
     | :math:`N`             | Trajectory index dimension                      |
     +-----------------------+-------------------------------------------------+
-    | :math:`[T]`           | Variable-length time dimension of each          |
-    |                       | trajectory                                      |
+    | :math:`T`             | Maximum length of a trajectory                  |
     +-----------------------+-------------------------------------------------+
     | :math:`S^*`           | Single-step shape of a time-series tensor       |
     +-----------------------+-------------------------------------------------+
-    | :math:`N \bullet [T]` | A dimension computed by flattening a            |
-    |                       | variable-length time dimension :math:`[T]` into |
-    |                       | a single batch dimension with length            |
-    |                       | :math:`sum_{i \in N} [T]_i`                     |
-    +-----------------------+-------------------------------------------------+
 
     Attributes:
+        paths (list[dict[str, np.ndarray or dict[str, np.ndarray]]]):
+            Nonflatten original paths from sampler.
```

Comment:
Wait, why is this necessary? It kinda defeats the point of the datastructure, and doesn't seem to be read anywhere.